.github/workflows/ci.yml

```diff
@@ -0,0 +1,115 @@
+name: CI - Test Runner
+
+# Run the workflow when commits are pushed on main or when a PR is modified
+on:
+  push:
+    branches:
+      - main
+
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+
+jobs:
+  ci:
+    name: CI
+    # Execute the CI on the course's runners
+    runs-on: ubuntu-latest
```

Comment:
I wonder if this still applies here since this repo is not part of the courses Github Organization and there are no runners configured for this project.
I'm not familiar with GitHub Actions, only with GitLab CI/CD and corresponding Runners... It's weird to me that this is running even though there are no Runners configured

Reply:
I just checked the [docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#choosing-github-hosted-runners) and `ubuntu-latest` is a GitHub-hosted runner. We should remove the comment, it's just confusing...

Reply:
Ohh, thanks for the clarification!